### ISSUE TITLE
'gradle release' task gives the error 'Server SSL certificate verification failed' for SVN

### ISSUE THREAD

**Issue by rmanders** [researchgate/gradle-release#201]
When I run the ```gradle build release``` command I get the following command line error output:

```
Running [svn, status, -q, -u] produced an error: [svn: E170013: Unable to connect to a repository at URL 'https://myurl.com/repo'
svn: E230001: Server SSL certificate verification failed: issuer is not trusted]
Running [svn, info, https://myurl.com/repo] produced an error: [svn: E170013: Unable to connect to a repository at URL 'https://myurl.com/repo'
svn: E230001: Server SSL certificate verification failed: issuer is not trusted]
```

When I run those commands directly from the command line:

```
%> svn status -q -u
%> svn info https://myurl.com/repo
```

There is no error and it works fine. The certificate is not a self-signed certificate and works with no problem in the browser.

I also followed the suggestions from this [stackoverflow article](http://stackoverflow.com/questions/3147660/server-certificate-verification-failed-issuer-is-not-trusted). Running the suggested commands did not prompt me for any kind of certificate verification and also worked without issue. However, I still cannot get rid of that error on running the release task.
 
Svn Version: 1.9.4 (r1740329)
Gradle Version: (tried with 2.13 and  3.1)
gradle-release version: 2.4.1

**Comment by rmanders** [comment]
I believe I found the cause of this issue. 

The SSL certificate was recently re-issued and my gradle was using an old cached certificate. Subversion was using the correct certificate (the newest one) which is why typing in svn commands on the CLI worked fine. 

I wasn't sure where gradle was caching the certificate so I first tried to delete ~/.gradle/caches but that had no effect. Finally, I deleted the entire ~/.gradle directory that that fixed the issue.

So in summary it appears that an old certificate for the svn repo was being cached by gradle, despite a new one being issued. Removing that fixed the issue.
